common4j/src/main/com/microsoft/identity/common/java/authorities/AzureActiveDirectoryAudience.java

```diff
@@ -154,6 +160,25 @@ public void setTenantId(String tenantId) {
         mTenantId = tenantId;
     }
 
+    public static String getAudienceFromTenantId( final String tenantId) {
+
+        if (tenantId == null) {
+            return UNKNOWN_AUDIENCE;
+        }
+
+        final String audienceToCheck = tenantId.toLowerCase(Locale.ROOT);
+
+        if (audienceToCheck.contains(CONSUMERS) || audienceToCheck.contains(MSA_MEGA_TENANT_ID)) {
```

Comment:
Instead of  `contains` why not do an `equals` comparison?

Reply:
[fbe8a84](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2495/commits/fbe8a841d3d18134489e28e519829cdbeb0fdc98)